crates/uv/src/main.rs

```diff
@@ -632,15 +632,15 @@ struct VenvArgs {
     /// The Python interpreter to use for the virtual environment.
     ///
     /// Supported formats:
-    /// * `-p 3.10` searches for an installed Python 3.10 (`py --list-paths` on Windows, `python3.10` on Linux/Mac).
+    /// - `3.10` searches for an installed Python 3.10 (`py --list-paths` on Windows, `python3.10` on Linux/Mac).
     ///   Specifying a patch version is not supported.
-    /// * `-p python3.10` or `-p python.exe` looks for a binary in `PATH`.
-    /// * `-p /home/ferris/.local/bin/python3.10` uses this exact Python.
+    /// - `python3.10` or `python.exe` looks for a binary in `PATH`.
+    /// - `/home/ferris/.local/bin/python3.10` uses this exact Python.
```

Comment:
Can we toggle the comment with cfg to have it be platform relevant? I think that'd make it a lot shorter and it wouldn't need to be listed like this?

Reply:
(Happy to track this in a follow-up too)

Reply:
Hmm, no if `cfg` applies to comments. We could define a custom struct for the `Path` and have two different definitions for each platform but that's a bit more involved (and requires copying the configuration)